ui/app/src/components/ICEToolsPanel/ICEToolsPanel.tsx

```diff
@@ -66,6 +67,16 @@ export function ICEToolsPanel() {
       width={width}
       maxWidth={windowSize}
       onWidthChange={onWidthChange}
+      onResizeStart={() => {
```

Comment:
@rart the issue is that in some browsers the preview iframe causes the mouse events not to trigger (while dragging the drawer). One one side I think this can be a preview util (to set/unset pointer events), but on the other side feels a bit weird to have a util to set pointer events. What do you think?
\* I tested this on chrome for macOS and the behavior is still fine

Reply:
Yes, mostly because the code is duplicated it needs refactoring.